mkosi.conf.d/fedora/10-fedora.conf

```diff
@@ -92,6 +92,7 @@ BuildPackages=
         pkgconfig(valgrind)
         pkgconfig(xencontrol)
         pkgconfig(xkbcommon)
+        python3-docutils
```

Comment:
Also here. `python3dist(docutils)`??

Reply:
Same thing, we just need the binaries so there's no point in using `python3dist()` here.